Framework/PythonInterface/test/python/plugins/algorithms/LoadWANDSCDTest.py

```diff
@@ -63,6 +67,19 @@ def test(self):
 
         LoadWANDTest_ws.delete()
 
+    def test_withNorm(self):
+        # create van data
+        van = LoadWANDSCD('HB2C_7000.nxs.h5')
+        van.setSignalArray(np.full_like(van.getSignalArray(), 25))
+        van.setErrorSquaredArray(np.full_like(van.getSignalArray(), 25))
+        AddSampleLog(van, LogName='time', LogText='42', LogType='Number Series', NumberType='Double')
+        AddSampleLog(van, LogName='monitor', LogText='420', LogType='Number Series', NumberType='Double')
+
+        LoadWANDTest_ws = LoadWANDSCD(Filename='HB2C_7000.nxs.h5,HB2C_7001.nxs.h5', VanadiumWorkspace=van, NormalizedBy='Monitor')
+        self.assertEqual(LoadWANDTest_ws.getSignalArray().max(), 7 / 25)
```

Comment:
You can also check that `getErrorSquaredArray()` is correct.

Reply:
got it, adding it now